engine/src/main/java/org/terasology/engine/subsystem/lwjgl/LwjglGraphics.java

```diff
@@ -254,22 +255,87 @@ public Subtexture buildAsset(AssetUri uri, SubtextureData data) {
-                & GLContext.getCapabilities().OpenGL15
-                & GLContext.getCapabilities().GL_ARB_framebuffer_object
-                & GLContext.getCapabilities().GL_ARB_texture_float
-                & GLContext.getCapabilities().GL_ARB_half_float_pixel
-                & GLContext.getCapabilities().GL_ARB_shader_objects;
-
-        if (!canRunGame) {
-            String message = "Your GPU driver is not supporting the mandatory versions or extensions of OpenGL. Considered updating your GPU drivers? Exiting...";
-            logger.error(message);
-            JOptionPane.showMessageDialog(null, message, "Mandatory OpenGL version(s) or extension(s) not supported", JOptionPane.ERROR_MESSAGE);
-            throw new RuntimeException(message);
+        boolean[] requiredCapabilities = {
+                GLContext.getCapabilities().OpenGL11,
+                GLContext.getCapabilities().OpenGL12,
+                GLContext.getCapabilities().OpenGL14,
+                GLContext.getCapabilities().OpenGL15,
+                GLContext.getCapabilities().GL_ARB_framebuffer_object,
+                GLContext.getCapabilities().GL_ARB_texture_float,
+                GLContext.getCapabilities().GL_ARB_half_float_pixel,
+                GLContext.getCapabilities().GL_ARB_shader_objects};
+
+        String[] capabilityNames = {"OpenGL11",
+                                    "OpenGL12",
+                                    "OpenGL14",
+                                    "OpenGL15",
+                                    "GL_ARB_framebuffer_object",
+                                    "GL_ARB_texture_float",
+                                    "GL_ARB_half_float_pixel",
+                                    "GL_ARB_shader_objects"};
+
+        boolean canRunTheGame = true;
+        String missingCapabilitiesMessage = "";
+
+        for (int index = 0; index < requiredCapabilities.length; index++) {
+            if (!requiredCapabilities[index]) {
+                missingCapabilitiesMessage += "    - " + capabilityNames[index] + "\n";
+                canRunTheGame = false;
+            }
         }
 
+        if (!canRunTheGame) {
+            String completeErrorMessage = completeErrorMessage(missingCapabilitiesMessage);
+            logger.error(completeErrorMessage);
+            JOptionPane.showMessageDialog(null, completeErrorMessage, "Required OpenGL version(s) or extension(s) not supported", JOptionPane.ERROR_MESSAGE);
+            System.exit(1);
+        }
+    }
+
+    private String completeErrorMessage(String errorMessage) {
+
+        String completeErrorMessage = "\n" +
+                "\nThe following OpenGL versions/extensions are required but are not supported by your GPU driver:\n" +
+                "\n" +
+                errorMessage +
+                "\n" +
+                "GPU Information:\n" +
+                "\n";
+
+        if (GLContext.getCapabilities().OpenGL11) {
```

Comment:
I agree.

I just spent some time digging in LWJGL documentation and forum. I'm still not sure 100%, but I think if the line Display.create() in LwjglGraphics.initDisplay() succeeds, *at least* an opengl 1.1 context is available through the GLContext object.

So, in the next commit I've eliminated the checks regarding opengl 1.1 and simplified the code accordingly.